pom.xml

```diff
@@ -66,23 +66,23 @@
     <name>${extension.name} API</name>
     <description>Project GlassFish Enterprise JavaBean API</description>
     
-    <url>http://metro.java.net/</url>
+    <url>https://javaee.github.io/metro/</url>
```

Comment:
This doesn't seem right, this is not a metro project

Reply:
This does not have corresponding rpc project under javaee.github.io. Not sure, why did they use metro URL before. Shall I leave it to the old value?

Reply:
Remove the url.